lib/attachment_reporting.rb

```diff
@@ -24,7 +24,7 @@ def create_organisation_attachment_count_hash
       # the editions of these sections in version order to find unique PDF attachments and their
       # publication times.
       all_unique_document_ids_for_manual(manual).each do |document_id|
-        section_editions = SectionEdition.where(document_id: document_id).order(:version_number)
+        section_editions = SectionEdition.where(document_id: document_id).order([:version_number, :asc])
```

Comment:
The commit message implies that the correct syntax is `order(version_number: :asc)`, but here it has been changed to `order([:version_number, :asc])`. I'm not sure which is correct.